### PR TITLE
Update indexing script to be more user friendly

### PR DESCRIPTION
**Pull request by Wynndow**
1. Allow auth token helpers to match dev aliases

2. Update docstring to better reflect script use

### CODE REVIEW

File: dmscripts/helpers/auth_helpers.py [hunk 1]
```diff
@@ -2,9 +2,10 @@
 import subprocess
 import yaml
 
+DEV_ALIASES = ('dev', 'development', 'local')
 
 def get_auth_token(api, stage):
-    if stage == 'development':
+    if stage in DEV_ALIASES:
```
Comment (samuelhwilliams) [inline dmscripts/helpers/auth_helpers.py]: Might be good to throw in `stage.lower()`

File: scripts/index-services.py [hunk 2]
```diff
@@ -2,18 +2,21 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --frameworks=<frameworks> [options]
+    index-services.py <stage> --index=<index> --frameworks=<frameworks> [options]
 
-    --serial                                      Do not run in parallel (useful for debugging)
-    --index=<index>                               Search API index name [default: g-cloud-9]
+    <stage>                                       One of dev, preview, staging or production
+    --index=<index>                               Search API index name
```
Comment (samuelhwilliams) [inline scripts/index-services.py]: Maybe say that it should be of the format `<framework>-YYYY-MM-DD`.

Comment (Wynndow) [reply scripts/index-services.py]: Does it actually need to be though? For local, would it matter?

Comment (samuelhwilliams) [reply scripts/index-services.py]: It doesn't matter specifically.

Comment (Wynndow) [reply scripts/index-services.py]: Cool. Have updated.

**Pull request merged**